<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FifoSimulator</class>
 <widget class="QMainWindow" name="FifoSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FIFO Process Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(235, 235, 241);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="body_frame">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>450</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_title">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>20</y>
          <width>340</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: none;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#555753;&quot;&gt;FIFO Process Simulator&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QFrame" name="circular_progress_bar_base">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>320</width>
          <height>320</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="circular_progress">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>300</width>
           <height>300</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	border-radius: 150px;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.749 rgba(211, 215, 207, 0), stop:0.75 rgba(136, 138, 133, 255));
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="circular_bg">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>300</width>
           <height>300</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	border-radius: 150px;
	
	background-color: rgba(211, 215, 207,120);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QFrame" name="container">
         <property name="geometry">
          <rect>
           <x>25</x>
           <y>25</y>
           <width>270</width>
           <height>270</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	border-radius: 135px;
	background-color: rgb(211, 215, 207);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_cpu">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>40</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>25</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: none;
	font-size: 25pt;
	color: rgb(85, 87, 83);
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#555753;&quot;&gt;CPU &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_cpu_percentage">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>100</y>
            <width>121</width>
            <height>81</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	background-color: none;
	font-size: 40pt;
	color: rgb(85, 87, 83);
}
</string>
          </property>
          <property name="text">
           <string>0%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <zorder>circular_bg</zorder>
        <zorder>circular_progress</zorder>
        <zorder>container</zorder>
       </widget>
       <widget class="QLabel" name="label_queue">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>150</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#555753;&quot;&gt;Fila de processos:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_start_stop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color:  rgb(235, 240, 243);
	border: 0px solid;
	border-radius: 5px;
	background-position: center;
	background-color: rgb(85, 87, 83);
}

QPushButton:hover {
	border: 2px solid  rgb(235, 240, 243);
	
	background-color: rgb(85, 87, 83);
}

QPushButton:pressed {
	color: rgb(69, 69, 69);
	background-color: rgb(217, 237, 247);
}</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>330</y>
          <width>381</width>
          <height>77</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_in_exec">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#828282;&quot;&gt;Em Execução: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_in_exec_p">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#828282;&quot;&gt;PX&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ready">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#828282;&quot;&gt;Pronto(Próximo): &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ready_p">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#828282;&quot;&gt;PY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QProgressBar" name="process_bar">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>200</y>
          <width>50</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QProgressBar {
	color:  rgb(69, 69,69);
	border: 0px solid;
	border-radius: 5px;
	border-style: none;
	background-position: center;
	text-align: center;
	background-color: rgb(115, 210, 22);
}

QProgressBar::chunk {
	background-color: rgb(238, 238, 236);

}
</string>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="stats_frame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(211, 215, 207);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_team">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>10</y>
          <width>251</width>
          <height>131</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#555753;&quot;&gt;Team:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#555753;&quot;&gt;Ivo H. Trindade Silva&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#555753;&quot;&gt;Lucas de Mendonça Schneider&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#555753;&quot;&gt;Jomilson Franklin Ciriaco&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="group_box_cpu">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>151</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(46, 52, 54);
}</string>
        </property>
        <property name="title">
         <string>CPU</string>
        </property>
        <widget class="QLabel" name="lavel_cpu_time">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Tempo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_cpu_idle">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ocioso</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_cpu_busy">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Ocupado</string>
         </property>
        </widget>
        <widget class="QLabel" name="cpu_time_value">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
        <widget class="QLabel" name="cpu_idel_value">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
        <widget class="QLabel" name="cpu_basy_value">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="group_box_wait">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>10</y>
          <width>151</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	
	color: rgb(46, 52, 54);
}</string>
        </property>
        <property name="title">
         <string>Espera</string>
        </property>
        <widget class="QLabel" name="label_wait_avarage">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Médio</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_wait_min">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Minimo</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_wait_max">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Máximo</string>
         </property>
        </widget>
        <widget class="QLabel" name="wait_min">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
        <widget class="QLabel" name="wait_max">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>70</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
        <widget class="QLabel" name="wait_mean">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>xxxx</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
